Answers.md

```diff
@@ -1,5 +1,16 @@
-1.  Explain the differences between `client-side routing` and `server-side routing`.
-1.  What does HTTP stand for?
-1.  What does CRUD stand for?
-1.  Which HTTP methods can be mapped to the CRUD acronym that we use when interfacing with APIs/Servers.
-1.  Mention three tools we can use to make AJAX requests
\ No newline at end of file
+<!-- 1.  Explain the differences between `client-side routing` and `server-side routing`. -->
+    Client-side routing performs the routing on the users browser while server-side routing performs on every the request to the server.
+
```

Comment:
Yes, however, while this is correct, the most important difference that distinguishes client-side routing from server-side routing is that client-side routing loads everything that it needs to show all routes on the first load. Server side loads everything that it needs to show a specific route every time it loads. 

Because of this, client side rendering has a slow first load, but is fast on every load after.